libs/libarchfpga/src/read_fpga_interchange_arch.cpp

```diff
@@ -163,6 +234,220 @@ struct ArchReader {
             }
         }
     }
+
+    // Layout Processing
+    void process_layout() {
+        auto strList = ar_.getStrList();
+        auto tileList = ar_.getTileList();
+        auto tileTypeList = ar_.getTileTypeList();
+        t_grid_def grid_def;
+
+        grid_def.width = grid_def.height = 0;
+        for (auto tile : tileList) {
+            grid_def.width = std::max(grid_def.width, tile.getCol() + 1);
+            grid_def.height = std::max(grid_def.height, tile.getRow() + 1);
+        }
+
+        grid_def.grid_type = GridDefType::FIXED;
+        std::string name = std::string(ar_.getName());
+        if (name == "auto") {
```

Comment:
Does it means that FPGA interchangeable layout will not support auto-sizing layout as VPR does nowadays? Should we clarify it in documentation? And should we comment here?

Reply:
At the moment, the interchange specifies fixed layout only architectures, and allowing for auto-sizing might be doable to allow for experimentation on new archs, but for the time being we may just restrict it to disallow `auto` sizes.

Reply:
That is o.k. We just need to document it well then.

Reply:
Ok, added some in-code comments about this